code/src/components/Name.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable */
+import React from 'react';
+
+export const Name = ({ name, setName }) => {
+    const handleNameChange = (event) => {
+        const upCaseUserName = event.target.value.charAt(0).toUpperCase() + event.target.value.slice(1);
+        setName(upCaseUserName);
+};
+
+    return (
+        <form onSubmit={event => event.preventDefault()}>
+            <h2>Order your movie popcorn with us!</h2>
+            <p>What is your name?</p>
+            <input
+            id="name-input"
+            className="name-input" 
+            type="text" 
+            value={name}
+            placeholder="Enter name here"
+            maxLength={30}
+            onChange={handleNameChange}
+            aria-label="Insert your name"
+            required /> 
+        </form>
```

Comment:
Perhaps an if statement of some sort or conditional rendering could be implemented that if the input is empty, it alerts the user to enter their name.